src/sentry/search/events/filter.py

```diff
@@ -1687,30 +1687,30 @@ def _release_stage_filter_converter(self, search_filter: SearchFilter) -> Option
 
     def _release_filter_converter(self, search_filter: SearchFilter) -> Optional[WhereType]:
         """Parse releases for potential aliases like `latest`"""
-        values = reduce(
-            lambda x, y: x + y,
-            [
-                parse_release(
-                    v,
-                    self.params["project_id"],
-                    self.params.get("environment_objects"),
-                    self.params.get("organization_id"),
-                )
-                for v in to_list(search_filter.value.value)
-            ],
-            [],
-        )
 
-        operator_conversions = {"=": "IN", "!=": "NOT IN"}
-        operator = operator_conversions.get(search_filter.operator, search_filter.operator)
-
-        return self._default_filter_converter(
-            SearchFilter(
-                search_filter.key,
-                operator,
-                SearchValue(values),
+        if search_filter.value.is_wildcard():
```

Comment:
This differs from the [original implementation](https://github.com/getsentry/sentry/blob/1cce810d3aec8b59514b8d8dab716b097542d999/src/sentry/search/events/filter.py#L1009-L1013) where it explicitly checks for `latest`, though I think I like this more as it becomes cleaner to reserve additional keywords for release names.